public/app.js

```diff
@@ -0,0 +1,61 @@
+// GLOBAL APP CONTROLLER
+class Controller {
+    static setupEventListeners() {
+        let DOMstrings = UIController.getDOMstrings();
+        document.querySelector(DOMstrings.logoutBtn).addEventListener('click', Controller.ctrlLogoutDashboard);
+        document.querySelector(DOMstrings.codingProgressLinkSelector).addEventListener('click', Controller.navigateToCodingProgress);
+        document.querySelector(DOMstrings.projectMenu).addEventListener('click', Controller.navigateToSelectedProject);          
+    };
+
+    static resetDashboard() {
+        let DOMstrings = UIController.getDOMstrings();
+        document.querySelector(DOMstrings.codingProgressContainer).innerHTML = "";
+        document.querySelector(DOMstrings.graphContainer).innerHTML = "";
+    }
+    
+    static ctrlLogoutDashboard() {
+        AuthController.logout()
+    };
+
+    static navigateToCodingProgress(e) {
+        Controller.resetDashboard()
```

Comment:
Should this be inside the if() statement?

Reply:
updated